commands/members/remove.js

```diff
@@ -2,24 +2,69 @@
 
 let cli = require('heroku-cli-util')
 let co = require('co')
+let Utils = require('../../lib/utils')
 
 function * run (context, heroku) {
-  let org = context.org
+  let orgInfo = yield Utils.orgInfo(context, heroku)
+  let groupName = context.org || context.flags.team
+  let teamInviteFeatureEnabled = false
+  let isInvitedUser = false
   let email = context.args.email
 
-  let request = heroku.request({
-    method: 'DELETE',
-    path: `/organizations/${org}/members/${encodeURIComponent(email)}`
-  })
-  yield cli.action(`Removing ${cli.color.cyan(email)} from ${cli.color.magenta(org)}`, request)
+  let teamInvites = function * () {
```

Comment:
This is repeated a few times. Can you move this into a helper, utility, or model to reduce the repetition?